src/detection/auto_calibration.py

```
@@ -0,0 +1,335 @@
+    adj_end = start_t + (end_t - start_t) / SLOW_FACTOR
+
+    start_f = int(adj_start * fps)
+    end_f = int(adj_end * fps)
+    cap.set(cv2.CAP_PROP_POS_FRAMES, start_f)
+
+    pupil_list = []
+    cr_list = []
+
+    print(f"=== {idx+1}/9 점 ({sx}, {sy}) ===")
+
+    for f in range(start_f, end_f):
```

Ruff (0.12.2)

Loop control variable `f` not used within loop body

Rename unused `f` to `_f`

(B007)